module/Entra/AdditionalFunctions/New-EntraOauth2PermissionGrant.ps1

```diff
@@ -0,0 +1,63 @@
+function New-EntraOauth2PermissionGrant {
+    [CmdletBinding(DefaultParameterSetName = 'InvokeByDynamicParameters')]
```

Comment:
The name of a parameter set should ideally tell what is specific for the set of parameters and why it is different than other sets. (The parameter set name is visible when the user runs `Show-Command <command>`).

`InvokeByDynamicParameters` doesn't do that.